features/front_office/request_edit_link.feature

```diff
@@ -0,0 +1,26 @@
+@email @edit_link
+Feature: Initiate edit/deregistration self-serve
+
+ As a waste exemption holder
+  I want to be able to enter my registration reference and emil address
+  So I receive a valid edit magic link by email
+  If the reference and email address don't match, no email will be sent.
+
+Scenario: Registration reference and email address do not match
+  When I start the flow to request an edit link
+   And I enter an invalid combination of reference number and email address
+  Then I will see the standard conditional confirmation page
+   And I will not receive an edit link email
+
+Scenario: Registration is not within the renewal window
+  Given I have a registration "outside" of the renewal window
+   When I start the flow to request an edit link
+    And I enter a valid combination of reference number and email address
```

Comment:
Should we not have `I will see the standard conditional confirmation page` here and in the below spec as well?

Reply:
This is in the flow to request an dit link via email, not the flow to actually perform edits.